src/interfaces.ts

```diff
@@ -96,6 +96,7 @@ export interface BCSettings {
   showAll: boolean;
   showGrid: boolean;
   showImpliedRelations: boolean;
+  showUpInJuggl : boolean  ; 
```

Comment:
I'm not seeing a default value? Should probably be false.

Reply:
yes it is false originally.